bentoml/cli/bento_management.py

```diff
@@ -172,23 +173,21 @@ def list_bentos(
         _print_bentos_info(result, output)
 
     @cli.command(
-        help='Delete bentos. To delete a bento service use "--name" and '
-        '"--version" options: "bentoml delete --name IrisClassifier '
-        '--bento-version 0.1.0" or "--tag" option: "bentoml delete --tag '
-        'IrisClassifier:v0.1.0". To delete multiple bentos, use "--name", "--tag", '
-        'and/or "--labels" to filter. To delete all bento services use "--all" option'
+        help='Delete bentos. Delete individual bento bundle: "bentoml delete '
+        'IrisClassifier:v1". Delete bento bundles with the same bento service name '
+        '"bentoml delete Irisclassifier". Delete multiple bento bundles: "bentoml '
+        'delete Irisclassifier:v1,ToxicCommentService:v123". Delete multiple bento '
+        'bundles using label filters: "bentoml delete --labels cicd=failed". Delete '
+        'all bento bundles "bentoml delete --all"',
     )
-    @click.option(
-        '--all', is_flag=True, help='Use this flag to remove all BentoServices'
-    )
-    @click.option(
-        '--tag',
+    @click.argument(
+        'tag',
```

Comment:
following the previous comment, can we rename this to `delete_target`? and add help message here:

```
Specify target service bundles to remove:
* Delete single service bundle by "name:version", e.g: `bentoml delete IrisClassifier:v1`
* Bulk delete all service bundle with a specific name, e.g.: `bentoml delete IrisClassifier`
* Bulk delete multiple service bundle by name and version, separated by ",", e.g.: `benotml delete Irisclassifier:v1,MyPredictService:v2`
* Bulk delete by tag, e.g.: `bentoml delete --tag`
* Buld delete all, e.g.: `bentoml delete --all`

```